.github/scripts/.helm-tests/default/result.yaml

```diff
@@ -241,6 +244,9 @@ spec:
               raw:
                 description: Raw contains the raw result of the SLO computation
                 type: string
+              state:
+                description: State describes the current state of the Analysis
```

Comment:
we should probably document allowed states here 😸

Reply:
allowed ? This is status, it's filled only by the controller

Reply:
Sure what I mean is to specify what the user can expect to see there: eg. Pending if not yet processed, Completed if done etc

Reply:
For instance, since we have this retry logic in case some retrieval from the provider fails the progressing stage may be interesting to document, so that user can decide wheter the error is gonna resolve itself or she needs to do something ex. fix improper provider setup